### ISSUE TITLE
[snap] /snap/multipass/x7/bin/launch-multipassd: mkdir: not found

### ISSUE THREAD

**Issue by townsend2010** [canonical/multipass#863]
I built a snap and installed it and now when `systemd` tries to start multipass, I get the following error:
```
multipass.multipassd[19006]: /snap/multipass/x7/bin/launch-multipassd: 7: /snap/multipass/x7/bin/launch-multipassd: mkdir: not found
```

I think this is caused by something in #835.

**Comment by ricab** [comment]
Well indeed `mkdir` is now used in the launchers, I am surprised it does not find it.

**Comment by townsend2010** [comment]
This was fixed by #873.
